client/payment-details/dispute-details/dispute-notice.tsx

```diff
@@ -70,6 +84,13 @@ const DisputeNotice: React.FC< DisputeNoticeProps > = ( {
 							href={ learnMoreDocsUrl }
 						/>
 					),
+					link: (
```

Comment:
💭 minor suggestion to clarify the difference with `a` and why it is needed: `inlineLink`?

Reply:
The class on the ExternalLink was causing a newline that I did not need, and I wrongly attributed it to ExternalLink. With the new message and formatting, I've switched back to using the existing `a` and `ExternalLink`